### DISCUSSION TITLE
Font size in Sun Valley tkinter ttk theme

### DISCUSSION THREAD

**Discussion by RedDoughnut** [rdbende/Sun-Valley-ttk-theme#87] [General]
I have a button that is a tab in ttk.Notebook and I used font size with ttk.Style but it doesn't work. Here is my code:
 
```
s = ttk.Style()
s.configure('normal.TButton', font=('Arial', 20))
findButton = ttk.Button(root1, text='Search', style='normal.TButton',command=search)
   
findButton.place(relx=0.805,rely=0.201)
```
Please somebody help!

Note:
- This is located inside a function
- I tried changing font (still not working)

If you have any questions ask.

**Comment by littlewhitecloud** [comment]
> I have a button that is a tab in ttk.Notebook and I used font size with ttk.Style but it doesn't work. Here is my code:
> 
> ```
> s = ttk.Style()
> s.configure('normal.TButton', font=('Arial', 20))
> findButton = ttk.Button(root1, text='Search', style='normal.TButton',command=search)
>    
> findButton.place(relx=0.805,rely=0.201)
> ```
> 
> Please somebody help!
> 
> Note:
> 
> * This is located inside a function
> * I tried changing font (still not working)
> 
> If you have any questions ask.

@Programmers3001 Try to do 
```python
root1.option_add('*font', ('Arial', 20))
```
